stress_testing/get.yml

```diff
@@ -13,7 +13,7 @@ config:
       arrivalRate: 100
       name: medium
     - duration: 60
-      arrivalRate: 150
+      arrivalRate: 1000
```

Comment:
If your service breaks somewhere between 100 requests per second and 1,000 requests per second, are you going to incrementally increase the RPS from 100 to 1,000 to isolate where exactly it breaks? example: test at 200 rps, 300 rps, and then you notice it breaks, so you test again at 225rps, 250 rps, and 275rps? This is what I plan on doing today but I am wondering what your plan is.